src/containers/SignUp/SignUp.test.js

```diff
@@ -39,42 +39,43 @@ describe('Signup', () => {
   })
 
   describe('handleSubmit', () => {
-    it('should sucessfully handle a new user signup', async () => {
+    it('should successfully handle a new user signup', async () => {
       const mockData = {
         id: 1,
         message: "New user created",
         status: "success",
       }
+      wrapper.instance().formRef = {
+        reset: jest.fn()
+      }
       API.postData = jest.fn().mockImplementation(() => {
         return mockData
       })
-      const expected = 'New user created'
+      const expected = 'New user created. Please log in.'
 
       await wrapper.instance().handleSubmit(mockPreventDefault)
 
       expect(wrapper.state('response')).toEqual(expected)
     })
 
-    it('should sucessfully handle a duplicate user signup', async () => {
-      const mockData = {
-        error: "Key email already exists."
-      }
+    it('should successfully handle a duplicate user signup', async () => {
       API.postData = jest.fn().mockImplementation(() => {
-        return mockData
+        return undefined
```

Comment:
We should look at this one together - I had a similar issue with some tests and I don't think they should be returning undefined. Can't remember if I figured it out.